modules/core/src/main/java/org/apache/ignite/internal/processors/platform/cache/PlatformCache.java

```diff
@@ -1422,6 +1422,11 @@ private Query readTextQuery(BinaryRawReader reader) {
         String typ = reader.readString();
         final int pageSize = reader.readInt();
 
+        //TODO uncomment when limit parameter is added to Platforms
```

Comment:
Let's either create a ticket for this or remove commented code.

https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-TODOs